modules/developers-guide/pages/graphql.adoc

```diff
@@ -1,6 +1,104 @@
 = GraphQL API
```

Comment:
Overall I find this page a bit frustrating, not your fault but because of the intrinsic complexity of having two different APIs. It's pretty complicated to explain and frankly I'm not sure I have a better suggestion 😞

It gets better once you branch into one of the dedicated "using" pages though. So maybe we should keep it short here, like "there are two ways to do things and here's where to find more". The sections on URIs and how to use the playground/postman are all repeated in the subpages, so I'm not sure if we need them here.

Reply:
I expanded with an example to compare the "schema" of each approach. I hope that will help with understanding.

I was thinking of have the sections on URIs and playground/postman only on this page (my preference), but wasn't sure if that was better, or I should have on each oriented page. I don't like the duplicating, either.